lib/daru/dataframe.rb

```diff
@@ -237,7 +237,7 @@ def from_vector_rows source, opts
     def initialize source, opts={} # rubocop:disable Metrics/MethodLength
       vectors, index = opts[:order], opts[:index] # FIXME: just keyword arges after Ruby 2.1
       @data = []
-      @name = opts[:name] || SecureRandom.uuid
+      @name = opts[:name]
```

Comment:
I'll just wait while you merge https://github.com/v0dro/daru/pull/147, and then rebase from master, and this PR would look OK :)